drivers/potentiometer/ad5293/README.rst

```diff
@@ -0,0 +1,159 @@
+====================
+AD5293 no-OS driver
+====================
+
+Supported Devices
+------------------
+
+`ad5293 <https://www.analog.com/en/products/ad5293.html>`_
+
+Overview
+---------
+Applications
+
+- Mechanical potentiometer replacement
+- Instrumentation: gain and offset adjustment
+- Programmable voltage-to-current conversion
+- Programmable filters, delays, and time constants
+- Programmable power supply
+- Low resolution DAC replacements
+- Sensor calibration
+
+Device Configuration
+--------------------
+
+1.  Serial peripheral interface (SPI) is necessary to access device
+2.  Daisy-chain mode is required for multiple chip use, in this configuration 
+    chip select line remains low untill all bytes are shifted out
+3.  High impedence mode of SDO pin is not recommanded, 
+    as tested somehow this could cause communications 
+    unfunctional
+
```

Comment:
also i think more details are needed in the documentation

i just searched a random device driver .rst to use as template:
https://github.com/analogdevicesinc/no-OS/blob/main/drivers/adc/adm1177/README.rst